<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>RegisterDialog</class>
  <widget class="QDialog" name="RegisterDialog">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>350</width>
        <height>300</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>用户注册</string>
    </property>
    <layout class="QVBoxLayout">
      <item>
        <widget class="QLabel">
          <property name="text">
            <string>新用户注册</string>
          </property>
          <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="txt_username">
          <property name="placeholderText">
            <string>请输入用户名</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="txt_password">
          <property name="placeholderText">
            <string>请输入密码</string>
          </property>
          <property name="echoMode">
            <enum>QLineEdit::Password</enum>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="txt_confirm">
          <property name="placeholderText">
            <string>确认密码</string>
          </property>
          <property name="echoMode">
            <enum>QLineEdit::Password</enum>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QPushButton" name="btn_register">
          <property name="text">
            <string>立即注册</string>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
</ui>